_data/history.yaml

```diff
@@ -0,0 +1,56 @@
+# Chronological history of the Nottingham New Theatre
+
+- year: 1927
+  title: Formation of Dramsoc
+  description: "The University College Nottingham Dramatic Society (Dramsoc) performed its first play in 1928: A.A. Milne's *Mr Pim Passes By*. <br /> The society offered members an opportunity to read, discuss, produce and enact plays."
```

Comment:
For all the description fields, I dislike the super long lines with <br /> in them. Can YML have multi-line fields?